pkgs/top-level/python-packages.nix

```diff
@@ -4114,6 +4114,7 @@ in {
   notify-py = callPackage ../development/python-modules/notify-py { };
 
   notmuch = callPackage ../development/python-modules/notmuch { inherit (pkgs) notmuch; };
+  notmuch2 = callPackage ../development/python-modules/notmuch/2.nix { inherit (pkgs) notmuch; };
```

Comment:
```suggestion
  notmuch = callPackage ../development/python-modules/notmuch { inherit (pkgs) notmuch; };

  notmuch2 = callPackage ../development/python-modules/notmuch/2.nix { inherit (pkgs) notmuch; };
```

Reply:
closely related packages tend to be clumped up together, look at neovim/xen/linux/win-qemu in all-packages.nix so I think it's fine.